### PR TITLE
Automl pytorch

### PR DESCRIPTION
**Pull request by LeonardPuettmann**

### CODE REVIEW

**Comment by FelixKirsch** [comment]
I have problems running the create_model script with the package versions specified in the requirements.txt. Updating the numpy version fixed it for me, although it is then not compatible with the embedders library.

File: ml/create_model.py [hunk 1]
```diff
@@ -208,39 +213,69 @@ def target_getter(df):
...
+for epoch in range(750):
+    y_hat = model(X_train)
+    loss = loss_function(y_hat, y_train)
+    losses.append(loss.item())
+
+    model.zero_grad()
+    loss.backward()
+
+    #if epoch % 10 == 0:
+    sys.stdout.write(f'\rLoss of {round(float(loss), 2)} at epoch {epoch} of 750')
```
Comment (FelixKirsch) [inline ml/create_model.py]: epoch + 1? otherwise it stops at 749 of 750

File: ml/create_model.py [hunk 2]
```diff
@@ -208,39 +213,69 @@ def target_getter(df):
...
+    nn.Linear(24, 3)
+).to(device)
+
+loss_function = nn.CrossEntropyLoss()
+optimizer = torch.optim.Adam(model.parameters())
+
+losses = []
+print(" ")
+print("Training model...")
+for epoch in range(750):
```
Comment (FelixKirsch) [inline ml/create_model.py]: i would suggest, saving the number of epochs in a variable, so you can easily change it later.
And also using this variable in the progress output

**Comment by FelixKirsch** [comment]
Running the container with the streamlit app does not work.
FileNotFoundError: [Errno 2] No such file or directory: '/automl/ml/model.pkl'
The created model is now called model.pt

File: ml/create_model.py (hunk 1 above)
Comment (LeonardPuettmann) [reply ml/create_model.py]: Good point. Changed epoch to epoch + 1

File: ml/create_model.py (hunk 2 above)
Comment (LeonardPuettmann) [reply ml/create_model.py]: Done!

File: requirements.txt [hunk 3]
```diff
@@ -1,10 +1,10 @@
 embedders==0.0.11
```
Comment (FelixKirsch) [inline requirements.txt]: updating embedders to version 0.0.14 solves conflicting dependencies with numpy

File: app/util.py [hunk 4]
```diff
@@ -6,7 +6,7 @@
 def get_model():
     global __loaded_model
     if not __loaded_model:
-        __loaded_model = pickle.load(open("/automl/ml/model.pkl", "rb"))
+        __loaded_model = pickle.load(open("/automl/ml/model.pt", "rb"))
```
Comment (FelixKirsch) [inline app/util.py]: use the torch package to load the model, atm the container fails while unpickling

**Comment by FelixKirsch** [comment]
You added an empty directory "automl-docker" to the repo. Is it used somewhere or was it by mistake?

**Review by FelixKirsch — APPROVED** [review]
looks good :)

**Pull request merged**